.changeset/tricky-glasses-try.md

```diff
@@ -0,0 +1,6 @@
+---
+"@graphiql/react": patch
+"graphiql": patch
+---
```

Comment:
This appears to be meant for humans to read and act on, with lines 2 and 3 being code, and lines 1 and 4 being code fences.  If that's correct, lines 1 and 4 should become triple-backticks ```` ``` ```` or triple-tildes `~~~` (whichever style you prefer), instead of triple-hyphens `---`
````suggestion
```
"@graphiql/react": patch
"graphiql": patch
```
````

Reply:
This is a changeset file, I'll leave it as is.

Reply:
@TallTed this file is machine read, the resulting changelog.md. entry is meant for humans to read

Reply:
@TallTed this file is machine read, the resulting changelog.md. entry is meant for humans to read